src/Surfnet/StepupMiddleware/MiddlewareBundle/Console/Command/MigrateInstitutionConfigurationsCommand.php

```diff
@@ -35,5 +40,74 @@ protected function configure()
 
     public function execute(InputInterface $input, OutputInterface $output)
     {
+        /** @var QuestionHelper $questionHelper */
+        $questionHelper = $this->getHelper('question');
+
+        $output->writeln([
+            '<error>WARNING: you are about to migrate institution configurations'
+            . ' to work with identifiers based on insitutions with normalized casing.</error>',
+            '<error>This command is intended to be run only once.</error>',
+            ''
+        ]);
+
+        $confirmationQuestion = new ConfirmationQuestion(
+            'Are you sure you want to run the institution configuration migrations? [y/N]',
+            false
+        );
+        $confirmed = $questionHelper->ask($input, $output, $confirmationQuestion);
+
+        if (!$confirmed) {
+            $output->writeln('<info>Exiting without running migrations.</info>');
+            return;
+        }
+
+        /** @var Container $container */
+        $container = $this->getApplication()->getKernel()->getContainer();
+
+        $tokenStorage     = $container->get('security.token_storage');
+        $connectionHelper = $container->get('surfnet_stepup_middleware_middleware.dbal_connection_helper');
+        $provider         = $container->get('surfnet_stepup_middleware_middleware.institution_configuration_provider');
+        $pipeline         = $container->get('pipeline');
+        $eventBus         = $container->get('surfnet_stepup_middleware_command_handling.event_bus.buffered');
+
+        // The InstitutionConfiguration commands require ROLE_MANAGEMENT
+        // Note that the new events will not have any actor metadata associated with them
+        $tokenStorage->setToken(new AnonymousToken('console', 'console', ['ROLE_MANAGEMENT']));
```

Comment:
This is necessary because otherwise the AuthenticationStage in the StagedPipeline would fail:
```
  [Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException]                                             
  The token storage contains no authentication token. One possible reason may be that there is no firewall configured for this URL. 
```

This happens because the InstitutionConfiguration commands, when checked for authorization, require a token to be present and require this token to have `ROLE_MANAGEMENT` as they are intended to be triggered from the Management API. Setting an (anonymous) token with that role mitigates that issue.